MANUAL_REGRESSION.md

```diff
@@ -0,0 +1,99 @@
+# Manual Regression Tests for JS SDK
+
+## Functional
+
+##### 1. Face photo webcam capture
+(on Google Chrome, Firefox, Safari, IE11 and Microsoft Edge browsers)
+
+0. Given webcam is connected to the computer
+1. Go through the flow to face capture
+    - browser should ask to enable the webcam
+2. Accept the webcam to be used on browser
+    - photo capture frame should display preview from webcam
+    - user should be able to upload a picture anyway
+3. Take photo with a webcam
+    - 3-second counter should start and then photo should be taken
+    - confirmation screen should show up containing photo that was taken
+    - user should be able to retake or continue with taken photo
+
+##### 2. Document photo webcam capture
+(on Google Chrome, Firefox, Safari, IE11 and Microsoft Edge browsers)
+
+0. Given webcam is connected to the computer
+1. Open link with additional GET parameter `?useWebcam=true`
+2. Go through the flow to document capture
+    - browser should ask to enable the webcam
+3. Accept the webcam to be used on browser
+    - photo capture frame should display preview from webcam
+    - user should be able to upload a picutre anyway
+4. Place document in front of camera so that it aligns the edges of document capture frame
+    - document should be auto-captured
+    - confirmation screen should show up containing a photo that was taken
+    - user should be able to retake or continue with taken photo
+
+##### 3. Cross-device with SMS
+(on one of the browsers)
+
+0. Given user is upload document page
```

Comment:
0. Given user is _on_ upload document page

Reply:
Done